ci/build_libtelio.py

```diff
@@ -373,6 +374,7 @@ def exec_bindings(args):
         try_download_artifacts(
             args.try_fetch_from_pipeline,
             PROJECT_ROOT,
+            PROJECT_ROOT,
```

Comment:
Both args are the same, it's confusing a bit

Reply:
Yeah, the idea is the `ArtifactsDownloader` to be re-usable, and those two parameters kind of makes sense.